src/main.rs

```diff
@@ -61,7 +62,13 @@ enum TargetGen {
 }
 
 fn main() -> Result<()> {
-    pretty_env_logger::init();
+    CombinedLogger::init(vec![TermLogger::new(
```

Comment:
This seems to cause the tests to fail in CI. Maybe we could just ignore errors here?

Reply:
Oh, I thought it failed to create the directory :) TermLogger is not available on all platforms/systems. I will feature gate it :)

Reply:
I think it depends on how the binary is run, the problem is probably that no proper terminal is available. I would prefer it to ignore errors, having to recompile everything because of this will be annoying.

Reply:
I am not sure I understand what you mean? We can just put a #[not(test)] ?